docs/syntax_and_semantics/c_bindings/enum.md

```diff
@@ -65,8 +65,8 @@ lib X
   end
 end
 
-X::SomeEnum::Zero # => 0_i8
-X::SomeEnum::Two  # => 2_i8
+puts X::SomeEnum::Zero.value # => 0_i8
+puts X::SomeEnum::Two.value  # => 2_i8
```

Comment:
**suggestion:** Remove `puts`. It seems counterproductive because it removes the type information: `puts 1_i8` and `puts 1_i32` have the same effect. 
The `.value` calls sure make sense to get the integer representation.